api/s2n.h

```diff
@@ -999,6 +999,12 @@ S2N_API extern int s2n_config_set_verify_host_callback(struct s2n_config *config
  *
  * The default value is 1 if the underlying libCrypto implementation supports OCSP. 
  *
+ * @note SHA-1 is the only supported hash algorithm for the `certID` field. If a different hash 
+ * algorithm is used, validation will fail and manual verification will be needed. This is 
+ * different from the hash algorithm used for the signature algorithm. See 
```

Comment:
nit - Might be a bit more clear:
```suggestion
 * different from the hash algorithm used for the OCSP signature. See 
```

Reply:
Also, it might be worth moving this sentence to immediately after your "only SHA-1 is supported" sentence to avoid confusion.